README.md

```
@@ -14,20 +18,32 @@
+- 🧩 **It supports multiple preview jobs.**
 
-- It is **free**.
-- It supports multiple preview jobs.
+## 📖 Table of Contents
 
-### Usage
+- [Usage](#-usage)
+  - [Multiple Jobs](#multiple-jobs)
+  - [Teardown](#teardown)
+  - [PRs from Forked Repositories](#-prs-from-forked-repositories)
+- [Inputs](#-inputs)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)